resolver/contentResolver.go

```diff
@@ -0,0 +1,70 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+)
+
+var serviceNameRegex = regexp.MustCompile("__(.*?)/")
+
+type contentResolver struct {
+	contentResolverAppURI  string
+	contentResolverAppName string
```

Comment:
aha, I didn't see that. Could you split the struct creation in `NewContentResolver` to multiple lines (a field per line) so that it's easier to follow.

I would have just gotten rid of the app name since you can easily tell from the url what app you are using. Also, when we move to k8s, that regex will no longer work since apps will no longer look like `__service-name/whatever`